modules/client/src/addresslistVoting/internal/graphql-queries/proposal.ts

```diff
@@ -28,7 +28,8 @@ query AddresslistVotingProposal($proposalId: ID!) {
     endDate
     executed
     earlyExecutable
-    potentiallyExecutable
+    approvalReached
+    isSignaling
```

Comment:
```suggestion
    isSignalling
```
Fix wrong spelling everywhere

Reply:
Both options are correct, US english tend not to double the l, I don't have a preference so I'll use the on you prefer